systemservices/marketplace/src/events/index.ts

```diff
@@ -0,0 +1,72 @@
+import { hexToAscii, fromUnit, findInAbi, parseTimestamp } from "../contracts/utils";
+import { ABIDefinition } from "web3/eth/abi";
+import _marketplaceABI from "../contracts/Marketplace.abi.json"
+const marketplaceABI = _marketplaceABI as ABIDefinition[]
+
+const eventHandlers: {[ethName: string]: {
```

Comment:
I'd rather to simplify this whole structure to something like below:

```js
export eventHandlers = {
  serviceCreated: () => {}, // handler func
  ...
}
```

We have the same event names for both smart contract and mesg with only the char case difference which we can dynamically convert to each other with a simple func.

This way readability for the code will be better and complexity will be avoided.